src/main/resources/application.yaml

```diff
@@ -34,6 +34,18 @@ spring:
           lob:
             # silence the 'wall-of-text' - unnecessary exception throw about blob types
             non_contextual_creation: true
+  mail:
+    host: false #smtp.office365.com
+    password: password
+    port: 587
+    properties:
+      mail:
+        smtp:
+          auth: true
+          starttls:
+            enable: true
+    test-connection: true
```

Comment:
Not feeling that strong but these are internally reports do we really want to fail the application is the test connection fails ?

Reply:
Provided possible config early. This in particular was to get feedback ;)